tests/test_humidity_sensor.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import absolute_import
+import unittest
+
+import mock
+
+from greenpithumb import humidity_sensor
+from greenpithumb import dht11_exceptions
+
+class HumiditySensorTest(unittest.TestCase):
+
+	def setUp(self):
+		self.mock_dht11 = mock.Mock()
+		self.humidity_sensor = humidity_sensor.HumiditySensor(
+			self.mock_dht11)
+
+	def test_humidity_50(self):
+		"""Value given should be value returned."""
+
```

Comment:
We generally shouldn't start methods with blank lines. I use blank lines in the places where the style guide mandates it, but other than that I try to limit blank lines so that I only use them to logically group blocks of code.

>Two blank lines between top-level definitions, be they function or class definitions. One blank line between method definitions and between the class line and the first method. Use single blank lines as you judge appropriate within functions or methods.

https://google.github.io/styleguide/pyguide.html?showone=Blank_Lines#Blank_Lines